gulpfile.js

```diff
@@ -139,7 +105,7 @@ gulp.task('tasks:new:ts', ['npminstall'], () =>
     .pipe(gulp.dest(paths.build.extensions.root))
 );
 
-gulp.task('tasks:new:common:ts', ['npminstall'], () => {
+gulp.task('tasks:new:common:ts', gulp.parallel('npminstall'), () => {
```

Comment:
What's the effect of `gulp.parallel` on a single task ?

Reply:
Nothing, I guess. It will just trigger a single task anyway. Could be series as well. It's the result of a search/replace I did in the code 🤷‍♂. I didn't find what the best practice is when depending on a single task.

Reply:
Ahh ok, so that's because you can not pass just a single task name anymore ?

Reply:
No. You have to pass `parallel()` or `series()`. Again, it's because they're deprecating `task()`, but they probably won't remove it before Gulp 5. So we have a few years ahead of us 🙂 .